sunway-college-johor-bahru.csl

```diff
@@ -149,10 +152,16 @@ TO DO
   </macro>
   <macro name="title">
     <choose>
-      <if type="chapter paper-conference article-journal article-magazine article-newspaper broadcast interview manuscript map personal_communication speech thesis map post post-weblog" match="none">
+    <!-- encyclopedia type  newspaer-->
+	  <if type="entry-encyclopedia article-newspaper">
```

Comment:
even if this may currently work correctly in Zotero or Mendeley, this isn't right. you want
`<if type="entry-encyclopedia article-newspaper" match="any">`
otherwise, the match condition should default to match="all", which in this case is impossible (and item can't be two item types at the same time.